Source/AwsCommonRuntimeKit/mqtt/MqttRequestResponseClient.swift

```diff
@@ -96,10 +111,17 @@ public struct MqttRequestResponse: Sendable {
 /// A response path is a pair of values - MQTT topic and a JSON path - that describe where a response to
 /// an MQTT-based request may arrive.  For a given request type, there may be multiple response paths and each
 /// one is associated with a separate JSON schema for the response body.
-public class ResponsePath: CStruct, @unchecked Sendable {
+final public class ResponsePath: CStruct, @unchecked Sendable {
```

Comment:
Trivial: Other areas you've used @unchecked explains why it's unchecked. Should it also be explained here?